src/components/JfListMakerComponent.spec.js

```diff
@@ -1,50 +1,389 @@
-import {shallowMount} from '@vue/test-utils';
+import {shallowMount, createLocalVue} from '@vue/test-utils';
 import JfListMakerComponent from './JfListMakerComponent.vue';
 
+/* Stubs custom directive */
+const localVue = createLocalVue();
+localVue.directive('jf-tooltip-on-overflow', {
+    bind: function() {}
+});
+
+// Utility method to avoid code duplication
+const mountComponent = function( props ){
+    let options = {
+        localVue,
+        propsData: Object.assign({}, props),
+        stubs: ['jf-field','jf-ui-select']
```

Comment:
when you shallowMount, vue will stub automatically all child components. You can consider to remove manual stubs.

Reply:
@ur92 it doesn't do so for components registered as global

Reply:
Oh interesting! Didn't know that... That could explain few issues that I have.